loaders/src/glTF/babylon.glTFFileLoader.ts

```diff
@@ -193,7 +194,7 @@ module BABYLON {
             mat = Matrix.Compose(scale, rotation, position);
         }
         else {
-            mat = Matrix.FromArray(node.matrix);
+            mat = Matrix.FromArray(node.matrix || [1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0]);

```

Comment:
`node.matrix ? Matrix.FromArray(node.matrix) : Matrix.Identity()` is probably a bit easier